app/src/main/java/com/teamagam/gimelgimel/app/view/viewer/data/entities/EntityBuilder.java

```diff
@@ -0,0 +1,50 @@
+package com.teamagam.gimelgimel.app.view.viewer.data.entities;
+
+import com.teamagam.gimelgimel.app.view.viewer.data.geometries.Geometry;
+import com.teamagam.gimelgimel.app.view.viewer.data.symbols.Symbol;
+
+/**
+ * Created by Yoni on 3/15/2016.
+ */
+public abstract class EntityBuilder<B extends EntityBuilder<B,E>,  E extends AbsEntity>{
+
+    public static final String LOG_TAG = EntityBuilder.class.getSimpleName();
+
+    protected static long numId = 0;
+    protected String mId;
```

Comment:
Use convetion for static members with prefixed s and lowerCamelCase.
Should be private.